cmd/integration-test/main.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"go.einride.tech/can/pkg/candevice"
+)
+
+const (
+	bitrate125K = 125000
+	bitrate250K = 250000
+)
+
+func main() {
```

Comment:
Why is the integration test created as a CLI with main and not as a standard Go test? It can be run as a normal Go test, or if needed it can leverage the main test functionality: https://pkg.go.dev/testing#hdr-Main

Reply:
If you feel limited by the CI (that it wants to run your integration tests) you can also add a build tag for integration tests, a quiet common technique. It's described here for example: https://peter.bourgon.org/go-in-production/#testing-and-validation